README.md

```diff
@@ -42,10 +42,10 @@ As an open source project, Ballerina welcomes contributions from the community.
 
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md).
 
-# Code of Conduct
+# Code of conduct
```

Comment:
Recheck heading levels with the guideline docs.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef